zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -0,0 +1,195 @@
+::
+
+  ZIP: XXX
+  Title: Proportional Output Fee Mechanism (POFM)
+  Owners: Aditya Bharadwaj <nighthawk24@gmail.com>
+  Credits: Madars Virza, Kris Nuttycombe
+  Status: Draft
+  Community Status: Request for comments : https://forum.zcashcommunity.com/t/zip-proportional-output-fee-mechanism-pofm/42808
+  Category: Wallet
+  Created: 2022-08-15
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED",
+"OPTIONAL", and "REQUIRED" in this document are to be interpreted as
+described in RFC 2119. [#RFC2119]_
+
+"Z" refers to shielded address.
+"T" refers to transparent address.
+
+"POFM" refers to the proposed change in conventional fee as described in 
+this ZIP.
+
+The term "default transaction fee" in this document is in reference
+to Z to Z, T to Z & Z to T transaction fee paid to miners on the Zcash network
+for the work on including the shielded transaction in a block.
```

Comment:
https://zips.z.cash/zip-0313#support suggested that we would try to make the standard fee also apply to fully transparent transactions (see also https://github.com/zcash/zcash/issues/2942). I realize that there are some transparent-only wallets that may be slow to implement this, if they implement it at all, but those wallets simply would not implement the ZIP and that's okay.

In particular, I think we should specify that *all* transactions that pay the standard fee regardless of transparency SHOULD be relayed, and also will be mined by miners using zcashd's `getblocktemplate` with the default config. That would pave the way for wallets to safely switch to using the standard fee for all transactions. (In practice this is likely to be a fee increase, which might be necessary in future if there is a DoS attacker focussing on fully transparent transactions.)